### PR TITLE
Last touches

### PR DESCRIPTION
**Pull request by Mateusszzp**

### CODE REVIEW

File: src/Features/People/Profile/profileSlice.js
Comment (HaniaNassalska) [inline src/Features/People/Profile/profileSlice.js:1]: I'm not sure about the slice method (crew, cast) ;)

**Review by HaniaNassalska — COMMENTED** [review]
Movie - cast and movie - crew - layout should look better ;) Everything else looks really nice! We are close to the end!

File: src/Features/People/Profile/profileSlice.js
Comment (Mateusszzp) [reply src/Features/People/Profile/profileSlice.js:1]: ok it can stay without it is not required but it looks better in my opinion

**Review by HaniaNassalska — COMMENTED** [review]
We almost have it! Credits - layout needs to be changed. Movie tile - rating should be rounded.  ;)

**Review by HaniaNassalska — APPROVED** [review]

**Pull request merged**